ipld/read_test.go

```diff
@@ -109,6 +109,8 @@ func TestBlockRecovery(t *testing.T) {
 }
 
 func TestRetrieveBlockData(t *testing.T) {
+	t.Skip("skipping as it spawns too many goroutines")
```

Comment:
Alright. It is not easily possible to only skip the large one, right?

Reply:
Possible

Reply:
Let's do that then :)

Reply:
Done

Reply:
Thanks for putting this in a separate PR btw 🙏🏼